src/icons/general/IconAdd.js

```diff
@@ -13,17 +13,10 @@ type Props = {
 };
 
 const IconAdd = ({ className = '', color = '#000000', height = 24, title, width = 24 }: Props) => (
-    <AccessibleSVG
-        className={`icon-add ${className}`}
-        height={height}
-        title={title}
-        viewBox="-603 389 16 16"
-        width={width}
-    >
+    <AccessibleSVG className={`icon-add ${className}`} height={height} title={title} viewBox="0 0 16 16" width={width}>
```

Comment:
Good opportunity to update to SUIT? `bdl-IconAdd`.

Reply:
Doesn't that make this a breaking change?

Reply:
Yep, it does, but it needs to get done eventually. The question is if it's something we can tackle now or not.

Reply:
yeah, but that should be fine because we have a lot of breaking changes going into 11.0 once we release it not as a beta.